nupic/research/frameworks/pytorch/model_utils.py

```diff
@@ -55,6 +56,10 @@ def train_model(
     :param batches_in_epoch: Max number of mini batches to train.
     :param device: device to use ('cpu' or 'cuda')
     :type device: :class:`torch.device
+    :param freeze_params: Parameters to freeze at specified indices
+     - freeze_params[0] -> network module
```

Comment:
`params[0]` and `params[1]` not `freeze_params`

Reply:
merged your comments :)